File: src/test/resources/ru/vyarus/gradle/plugin/animalsniffer/java/invalid/Sample2.java
package invalid;

import org.slf4j.LoggerFactory;

import java.nio.file.FileSystems;

public class Sample2 {

    public static void main(String[] args) {
        // added in 1.7
        FileSystems.getDefault().getFileStores();
        LoggerFactory.getLogger(Sample2.class);
    }
}

File: src/test/resources/ru/vyarus/gradle/plugin/animalsniffer/kotlin/invalid/Sample2.kt
package invalid

import org.slf4j.LoggerFactory
import java.nio.file.FileSystems

public class Sample2 {

    fun main() {
        // added in 1.7
        FileSystems.getDefault().fileStores
        LoggerFactory.getLogger(Sample2::class.java)
    }
}